gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -66,24 +67,37 @@
  * host should actually complete its work while having the lease and then mark the flow action as NULL to indicate no
  * further leasing should be done for the event.
  */
-public class MySQLMultiActiveLeaseArbiter implements MultiActiveLeaseArbiter {
+@Slf4j
+public class MysqlMultiActiveLeaseArbiter implements MultiActiveLeaseArbiter {
   /** `j.u.Function` variant for an operation that may @throw IOException or SQLException: preserves method signature checked exceptions */
   @FunctionalInterface
   protected interface CheckedFunction<T, R> {
     R apply(T t) throws IOException, SQLException;
   }
 
-  public static final String CONFIG_PREFIX = "MySQLMultiActiveLeaseArbiter";
+  public static final String CONFIG_PREFIX = "MysqlMultiActiveLeaseArbiter";
 
   protected final DataSource dataSource;
   private final String leaseArbiterTableName;
   private final String constantsTableName;
   private final int epsilon;
   private final int linger;
+
+  // TODO: define retention on this table
+  private static final String CREATE_LEASE_ARBITER_TABLE_STATEMENT = "CREATE TABLE IF NOT EXISTS %S ("
+      + "flow_group varchar(" + ServiceConfigKeys.MAX_FLOW_GROUP_LENGTH + ") NOT NULL, flow_name varchar("
+      + ServiceConfigKeys.MAX_FLOW_GROUP_LENGTH + ") NOT NULL, " + "flow_execution_id varchar("
+      + ServiceConfigKeys.MAX_FLOW_EXECUTION_ID_LENGTH + ") NOT NULL, flow_action varchar(100) NOT NULL, "
+      + "event_timestamp TIMESTAMP DEFAULT CURRENT_TIMESTAMP, "
```

Comment:
suggest no default since probably shouldn't be possible to insert w/o a specific `event_timestamp`

(the default for `lease_acq_tstamp` seems reasonable)